lib/pasarguard-backup.sh

```
@@ -377,35 +400,44 @@ send_backup_error_to_telegram() {
-        response=$(curl "${curl_proxy_args[@]}" -s -w "%{http_code}" -o /tmp/tg_response.json \
-            -F chat_id="$BACKUP_TELEGRAM_CHAT_ID" \
-            -F document=@"$log_file;filename=backup_error.log" \
-            -F caption="📜 Backup Error Log - $error_time" \
-            "https://api.telegram.org/bot$BACKUP_TELEGRAM_BOT_KEY/sendDocument")
-
-        http_code="${response:(-3)}"
-        if [ "$http_code" -eq 200 ]; then
-            colorized_echo green "Backup error log sent to Telegram."
+        curl "${curl_proxy_args[@]}" -s -X POST "https://api.telegram.org/bot$BACKUP_TELEGRAM_BOT_KEY/sendMessage" \
+            -d chat_id="$chat_id" \
+            -d text="$message" >/dev/null 2>&1 &&
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)